src/__tests__/lib/item-input/array.test.ts

```diff
@@ -598,34 +602,37 @@ describe('checkboxDef', () => {
 
 		it('uses supplied validate method', async () => {
 			const validate = jest.fn<Required<CheckboxDefOptions<string>>['validate']>()
-			const def = checkboxDef<string>('Checkbox Def', ['Item 1', 'Item 2', 'Item 3'], { validate })
+			const def = checkboxDef('Checkbox Def', ['Item 1', 'Item 2', 'Item 3'], { validate })
```

Comment:
Why was `<string>` removed here?

Reply:
Because it was taking up space and making the line too long and TS could figure it out without it.